### PR TITLE
add svg logo [SKIP CI]

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

update logo for docs

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
ok, need to get the SVG versions :]

**Comment by Borda** [comment]
Turned out that the logo is used from template not from this repo, to be used from here needs a larger change, will resume later...

**Review by justusschock — APPROVED** [review]

File: docs/source/conf.py [hunk 1]
```diff
@@ -175,7 +175,7 @@
     'logo_only': False,
 }
 
-html_logo = '_images/logos/lightning_logo-name.svg'
+html_logo = '_images/logos/lightning_logo-name.png'
```
Comment (SkafteNicki) [inline docs/source/conf.py:178]: Why not the `svg` version?

**Review by SkafteNicki — APPROVED** [review]

**Review by ydcjeff — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -94,7 +94,7 @@ references:
 
 jobs:
 
-  TPU-tests:
+  build-test:
```
Comment (Borda) [inline .circleci/config.yml:97]:
```suggestion
  TPU-tests:
```

**Pull request merged**